### PR TITLE
Add a basic text FPS monitor

### PR DESCRIPTION
**Pull request by 7oxicshadow**
This adds a basic text fps monitor that sits in the top left of the screen.

As there is already a substantial fps/mspf monitor in xemu I have not exposed this in the UI.

Instead it can either be controlled by modifying the .ini directly or by opening the monitor window and typing showfps 0 or 1

image

### CODE REVIEW

**Comment by dracc** [comment]
Commit messages doesn't follow the format used in this repository (\<component>: \<what was done>).
Please consider rebasing on master instead of merging, makes for a cleaner history.

**Comment by 7oxicshadow** [comment]
My Bad, There was an option to resolve conflicts from within github. Never used it before. It only mentioned 2 files. When I actually resolved them it went rogue and did a full merge. Wasn't expecting that.

**Comment by simi** [comment]
@7oxicshadow can you post screenshot as well?

_Btw. @dracc it is possible to squash and merge later using GitHub UI._

**Comment by 7oxicshadow** [comment]
Screenshot added

File: ui/xemu-hud.cc [hunk 1]
```diff
@@ -2412,6 +2481,13 @@ void xemu_hud_render(void)
     ImGui::NewFrame();
     process_keyboard_shortcuts();
 
+    int ui_show_fps_bool;
+    xemu_settings_get_bool(XEMU_SETTINGS_DISPLAY_SHOW_FPS, &ui_show_fps_bool);
+    if(ui_show_fps_bool) {
+    	// Draw the FPS before the menu
+    	fps_manager.Draw();
```
Comment (GXTX) [inline ui/xemu-hud.cc:2488]: Missing spaces

**Comment by MasonT8198** [comment]
@mborgerson I believe the new UI takes care of this, however thanks for the work @7oxicshadow

**Comment by Adamillo** [comment]
@7oxicshadow what's the status of this PR?

**Comment by 7oxicshadow** [comment]
@Adamillo I think the general consensus is that this PR should be abandoned in favour of using the built in FPS/MSPF video window. I can understand why as it is a duplication of work.

I have recently created a PR to allow the FPS/MSPF window state to be remembered across restarts so this should hopefully achieve a similar goal to what this PR was intended for. If it is accepted it might need a follow up PR to allow the video window to automatically open at startup if it was open at previous shutdown to achieve parity.

I will close this pull request and it can be re-opened in the future should anyone decide that it is needed. I will keep my private branch alive so the work is not lost.

**Pull request closed without merging**